hapi/src/main/java/module-info.java

```diff
@@ -56,12 +56,14 @@
     exports com.hederahashgraph.api.proto.java;
     exports com.hederahashgraph.service.proto.java;
     exports com.hedera.hapi.util;
-    exports com.hedera.hapi.platform.state;
-    exports com.hedera.hapi.node.state.roster;
     exports com.hedera.hapi.block.stream;
     exports com.hedera.hapi.block.stream.input;
     exports com.hedera.hapi.block.stream.output;
+    exports com.hedera.hapi.platform.state;
+    exports com.hedera.hapi.node.state.roster;
     exports com.hedera.hapi.block.stream.schema;
+    exports com.hedera.hapi.block.protoc;
+    exports com.hedera.hapi.block.stream.protoc;
```

Comment:
Is there a chance we could avoid the protoc versions and use the PBJ versions only?

Reply:
Yes, the only offender was `GrpcBlockItemWriter` and since it was an unused WIP, just removed it along with these `exports`.